### DISCUSSION TITLE
dowód zakupu

### DISCUSSION THREAD

**Discussion by akxx87** [allegro/allegro-api#6376] [Suggestions - rejected]
Czy używając requesta opisanego tutaj: 
https://developer.allegro.pl/tutorials/jak-obslugiwac-zamowienia-GRaj0qyvwtR#jak-dodac-fakture-do-zamowienia

Można również dodawać dowód zakupu faktura imienna dla zwykłych kupujących, którzy nie wybrali FV przy zakupie ?

**Comment by MaciejFrackowiak** [comment]
Wątek zamykam i [przenoszę](https://github.com/allegro/allegro-api/issues/6378), ponieważ zakładka Discussion przeznaczona jest na sugestie.
